examples/delegator-generation/src/adder.json

```diff
@@ -1,38 +1,39 @@
 {
-  "abi_schema_version": "0.1.0",
+  "schema_version": "0.1.0",
   "metadata": {
     "name": "abi",
     "version": "0.1.0",
     "authors": [
       "Near Inc <hello@nearprotocol.com>"
     ]
   },
-  "abi": {
+  "body": {
     "functions": [
       {
         "name": "add",
+        "doc": " Adds two pairs point-wise.",
         "is_view": true,
         "params": [
           {
             "name": "a",
+            "serialization_type": "json",
```

Comment:
@itegulov what was the status of why the serialization type is added to each parameter? (Curious because seems redundant -- you can't mix serialization protocols for different params)

Reply:
Yeah I look into this a while ago but postponed it due to some uncertainty about how this should look and then it just slipped from my mind. Sorry about that, created an issue so we don't forget: https://github.com/near/near-abi-rs/issues/7. I don't think it is a big deal to stay like this till NEARCon though, wdyt?

Reply:
Agree :)